resources/l10n/ja/LC_MESSAGES/loot.po

```diff
@@ -301,657 +470,796 @@ msgstr ""
+#~ "most recent valid revision instead. Syntax errors are usually minor and "
+#~ "fixed within hours."
+#~ msgstr ""
+#~ "最新のリビジョンのマスターリストに構文エラーが含まれているため、直近の有効"
+#~ "リビジョンのマスターリストをかわりに使用します。基本的に構文エラーは小さな"
+#~ "不具合ですので、数時間内に修正されるでしょう。"
+
+#~ msgid "Cyclic interaction detected"
+#~ msgstr "循環参照を検出"
+
+#~ msgid "Failed to sort plugins. Details: %s"
+#~ msgstr "プラグインのソートに失敗しました。詳細: %s"
+
+#~ msgid "Redate"
+#~ msgstr "日付を変更する"
+
+#~ msgid "Clear"
+#~ msgstr "消去する"
+
+#~ msgid "Do not clean"
+#~ msgstr "クリーン禁止"
+
+#~ msgid "N/A"
+#~ msgstr "利用不可"
+
+#~ msgid "Master File"
+#~ msgstr "マスターファイル"
+
+#~ msgid "Light Master File"
+#~ msgstr "ライトマスターファイル"
+
+#~ msgid "Save Metadata"
+#~ msgstr "メタデータを保存"
+
+#~ msgid "Cancel"
+#~ msgstr "キャンセル"
+
+#~ msgid "A filename is required."
+#~ msgstr "ファイル名を入れてください。"
+
+#~ msgid "A content string is required."
+#~ msgstr "内容を書いてください。"
+
+#~ msgid "A CRC is required."
+#~ msgstr "CRCを入れてください。"
+
+#~ msgid "Values must be integers."
+#~ msgstr "整数にしてください。"
+
+#~ msgid "A utility name is required."
+#~ msgstr "ツール名を入れてください。"
+
+#~ msgid "A link is required."
+#~ msgstr "リンクを入れてください。"
+
+#~ msgid "Jump To General Information"
+#~ msgstr "トップに戻る"
+
+#~ msgid "Apply"
+#~ msgstr "適用"
+
+#~ msgid "Open Debug Log Location"
+#~ msgstr "デバッグログの保存先を開く"
+
+#~ msgid "Press Enter or click outside the input to set the filter."
+#~ msgstr ""
+#~ "Enterキーを押すか入力欄の外をクリックしてフィルターを確定してください。"
+
+#~ msgid "Enable Edits"
+#~ msgstr "編集を有効にする"
+
+#~ msgid "Masterlist Repository URL"
+#~ msgstr "マスターリストのリポジトリURL"
+
+#~ msgid "Masterlist Repository Branch"
+#~ msgstr "マスターリストのリポジトリブランチ"
+
+#~ msgid "Main"
+#~ msgstr "メイン"
+
+#~ msgid "Click %(menu_icon)s buttons to open menus."
+#~ msgstr "%(menu_icon)s ボタンをクリックするとメニューが開きます。"
+
+#~ msgid ""
+#~ "As well as messages, LOOT displays plugin <span class=\"version\">version "
+#~ "numbers</span>, <span class=\"crc\">CRCs</span> and Bash Tag suggestions "
+#~ "for <span class=\"tag add\">addition</span> and <span class=\"tag remove"
+#~ "\">removal</span>."
+#~ msgstr ""
+#~ "画面右のメインタブにはメッセージのほかに、プラグインの<span class="
+#~ "\"version\">バージョン番号</span>、<span class=\"crc\">CRC</span>、<span "
+#~ "class=\"tag add\">追加</span>または<span class=\"tag remove\">削除</span>"
+#~ "するべきBash Tagの候補などが表示されます。"
+
+#~ msgid ""
+#~ "Double-click a plugin in the sidebar to quickly open it in the metadata "
+#~ "editor."
+#~ msgstr ""
+#~ "画面左のサイドバー内のプラグインをダブルクリックするとメタデータ編集画面を"
+#~ "直接開くことができます。"
+
+#~ msgid ""
+#~ "Many interface elements have tooltips. If you don't know what something "
+#~ "is, try hovering your mouse over it to find out. Otherwise, LOOT's "
+#~ "documentation can be accessed through the main menu."
+#~ msgstr ""
+#~ "画面上の項目の多くにはヒントが用意されています。どういう機能か知りたくなっ"
+#~ "たらカーソルを載せて調べてみてください。また、メニューからLOOTのマニュアル"
+#~ "を開くこともできます。"
+
+#~ msgid "OK"
+#~ msgstr "OK"
 
 #~ msgid "Error: Could not initialise logging."
 #~ msgstr "エラー: ログの出力を開始できません。"
```

Comment:
Please delete these old & outcommented translations, using `Translation > Purge deleted translations > Purge` in Poedit.

Reply:
Thanks for the review. I have fixed those.